spec/index.html

```diff
@@ -1317,12 +1312,13 @@ <h3>Datatype IRIs</h3>
         refers to the datatype <code><a>rdf:JSON</a></code>.</li>
     </ul>
 
-    <p>RDF processors are not required to recognize datatype IRIs.
-      Any literal typed with an unrecognized IRI is treated just like
-      an unknown IRI, i.e. as referring to an unknown thing. Applications
-      MAY give a warning message if they are unable to determine the
-      referent of an IRI used in a typed literal, but they SHOULD NOT
-      reject such RDF as either a syntactic or semantic error.<p>
+    <p>RDF processors are not required to handle all datatypes.
+      Any literal typed with a datatype IRI not handled by an RDF processor
+      is treated just like an unknown IRI, i.e. as referring to an unknown thing.
+      Applications MAY give a warning message if they are unable to determine the
+      referent of an IRI used in a typed literal. RDF processors SHOULD
+      not reject literals with unknown IRI as either a syntactic or
+      semantic error.<p>
```

Comment:
```suggestion
    <p>RDF processors are not required to handle all datatypes.
      Any literal typed with a datatype IRI not handled by an RDF processor
      is treated just like an unknown IRI, i.e., as referring to an unknown thing.
      Applications MAY give a warning message if they are unable to determine the
      referent of an IRI used in a typed literal. RDF processors SHOULD NOT
      reject a literal with an unknown IRI as either a syntactic or
      semantic error.<p>
```

Reply:
There is a slight inconsistency in this paragraph. According to the first sentence, the things that RDF processors may handle are the datatypes. Yet, according to the second sentence, it is datatype IRIs that RDF processors may (or may not) handle. We should settle on one or the other (datatypes or datatype IRIs).

My preference would be for datatypes instead of datatype IRIs. In this case, the second sentence here should be change to: "Any literal typed with a datatype IRI that denotes a datatype that is not handled by ..."

Then, the new line 746 (see above) should be changed accordingly: "If the <a>datatype</a> denoted by the literal's <a>datatype IRI</a> is handled by ..."

Reply:
OK - I will make the change here and L746.

As for more general "datatype IRI" -> "datatype", please could we have a separate PR? By bundling changes into one PR, it becomes increasingly harder to ensure, and review, that it is done outside the areas touched by the initial PR.